cmd/podman/common.go

```diff
@@ -173,363 +127,367 @@ func getDefaultNetwork() string {
+	)
+	createFlags.Int64(
+		"cpu-rt-runtime", 0,
+		"Limit the CPU real-time runtime in microseconds",
+	)
+	createFlags.Uint64(
+		"cpu-shares", 0,
+		"CPU shares (relative weight)",
+	)
+	createFlags.Float64(
+		"cpus", 0,
+		"Number of CPUs. The default is 0.000 which means no limit",
+	)
+	createFlags.String(
+		"cpuset-cpus", "",
+		"CPUs in which to allow execution (0-3, 0,1)",
+	)
+	createFlags.String(
+		"cpuset-mems", "",
+		"Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.",
+	)
+	createFlags.BoolP(
+		"detach", "d", false,
+		"Run container in background and print container ID",
+	)
+	createFlags.String(
+		"detach-keys", "",
+		"Override the key sequence for detaching a container. Format is a single character `[a-Z]` or `ctrl-<value>` where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`",
+	)
+	createFlags.StringSlice(
+		"device", []string{},
+		"Add a host device to the container (default [])",
+	)
+	createFlags.StringSlice(
+		"device-read-bps", []string{},
+		"Limit read rate (bytes per second) from a device (e.g. --device-read-bps=/dev/sda:1mb)",
+	)
+	createFlags.StringSlice(
+		"device-read-iops", []string{},
+		"Limit read rate (IO per second) from a device (e.g. --device-read-iops=/dev/sda:1000)",
+	)
+	createFlags.StringSlice(
+		"device-write-bps", []string{},
+		"Limit write rate (bytes per second) to a device (e.g. --device-write-bps=/dev/sda:1mb)",
+	)
+	createFlags.StringSlice(
+		"device-write-iops", []string{},
+		"Limit write rate (IO per second) to a device (e.g. --device-write-iops=/dev/sda:1000)",
+	)
+	createFlags.StringSlice(
+		"dns", []string{},
+		"Set custom DNS servers",
+	)
+	createFlags.StringSlice(
+		"dns-opt", []string{},
+		"Set custom DNS options",
+	)
+	createFlags.StringSlice(
+		"dns-search", []string{},
+		"Set custom DNS search domains",
+	)
+	createFlags.String(
+		"entrypoint", "",
+		"Overwrite the default ENTRYPOINT of the image",
+	)
+	createFlags.StringSliceP(
+		"env", "e", []string{},
+		"Set environment variables in container",
+	)
+	createFlags.StringSlice(
+		"env-file", []string{},
+		"Read in a file of environment variables",
+	)
+	createFlags.StringSlice(
+		"expose", []string{},
+		"Expose a port or a range of ports (default [])",
+	)
+	createFlags.StringSlice(
+		"gidmap", []string{},
+		"GID map to use for the user namespace",
+	)
+	createFlags.StringSlice(
+		"group-add", []string{},
+		"Add additional groups to join (default [])",
+	)
+	createFlags.Bool(
+		"help", false, "",
+	)
+
+	createFlags.StringP(
+		"hostname", "h", "",
+		"Set container hostname",
+	)
+	createFlags.String(
+		"image-volume", "bind",
+		"Tells podman how to handle the builtin image volumes. The options are: 'bind', 'tmpfs', or 'ignore' (default 'bind')",
+	)
+	createFlags.Bool(
+		"init", false,
+		"Run an init binary inside the container that forwards signals and reaps processes",
+	)
+	createFlags.String(
+		"init-path", "",
```

Comment:
Should this be defaulted to libpod.DefaultInitPath?

Reply:
it isnt in master, which is what I followed. it **is** alter in create.go depending ...